### PR TITLE
Panel refactor

### PR DESCRIPTION
**Pull request by gazuntype**
just made some code edits. renamed variables etc etc.

### CODE REVIEW

**Review by chornesays — APPROVED** [review]
Prospective approval pending cdiddy signoff on magic consts

File: Editor/Authentication.cs [hunk 1]
```diff
@@ -24,21 +24,21 @@ public enum AuthenticationState {
...
-        public static Authentication Instance { get; private set; }
+        public static Authentication Instance { get;}
         static Authentication() {
             Instance = new Authentication();
         }
-        private const string REFRESH_KEY = "RefreshToken";
-        private const string customTokenLoginURL = "https://identitytoolkit.googleapis.com/v1/accounts:signInWithCustomToken?key=";
-        private const string refreshURL = "https://securetoken.googleapis.com/v1/token?key=";
-        private LoginResponse loginData;
+        private const string RefreshKey = "RefreshToken";
```
[the remark this answers is not in the document]
Comment (cdiddy77) [reply Editor/Authentication.cs:31]: I have long ago given up having a strong opinion in such matters, and as such, when in doubt, defaulting to the Microsoft StyleCop standard seems reasonable? As such, RefreshKey wins:
https://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants

File: Editor/Authentication.cs [hunk 2]
```diff
@@ -47,34 +47,30 @@ public class Authentication {
             }
         }
 
-        public bool IsLoggedIn {
-            get {
-                return AuthState == AuthenticationState.LoggedIn;
-            }
-        }
+        public bool IsLoggedIn => AuthState == AuthenticationState.LoggedIn;
```
[the remark this answers is not in the document]
Comment (cdiddy77) [reply Editor/Authentication.cs:50]: It's just a shorthand known as [__expression body definitions__](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties#expression-body-definitions)

**Review by cdiddy77 — APPROVED** [review]
Funny, @gazuntype I always took you for a free-spirit who eschews the hobgoblin of conformity. It almost looks like you used StyleCop....

File: Editor/Authentication.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (chornesays) [reply Editor/Authentication.cs:50]: i love it so much

File: Editor/Authentication.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (chornesays) [reply Editor/Authentication.cs:31]: StyleCop it is LFG

**Comment by gazuntype** [comment]
> Funny, @gazuntype I always took you for a free-spirit who eschews the hobgoblin of conformity. It almost looks like you used StyleCop....

i am a man of mystery.

**Pull request merged**